package.json

```diff
@@ -16,6 +16,7 @@
     "url": "https://github.com/Joystream/joystream/issues"
   },
   "scripts": {
+    "bundle-stats": "STATS=server npm run build",
```

Comment:
Does stats generation take some considerable time? I'm wondering whether maybe we should just enable it by default with a build?

Reply:
I don't think it adds much more time but this opens a server and a tab on the browser, the build does make a file by default called stats.json which can be used as input on https://chrisbateman.github.io/webpack-visualizer/
or to open the the bundle analizer from the cli